### PR TITLE
Pile of How-tos converted from GM2 parlance

### PR DESCRIPTION
**Pull request by theduckylittle**
Flushing out how-to documentation for the new version of GeoMoose.

### CODE REVIEW

**Comment by theduckylittle** [comment]
Dunno why Travis is so angry. Tests and lint are clean locally.

**Comment by klassenjs** [comment]
Because
```
tests/gm3/components/catalog.test.js
  ● Test suite failed to run
    /home/travis/build/geomoose/gm3/node_modules/ol/layer/base.js:42
      /**
      ^^^
    
    SyntaxError: Invalid or unexpected token
      
      at transformAndBuildScript (node_modules/jest-runtime/build/transform.js:321:12)
      at Object.<anonymous> (node_modules/ol/layer/layer.js:5:13)
      at Object.<anonymous> (node_modules/ol/layer/vector.js:2:14)
```

But I donno why a comment is a syntax error.

**Comment by theduckylittle** [comment]
Again, not sure why Travis is angry about that.  NOt sure how it managed to bungle the OpenLayers import.

**Comment by klassenjs** [comment]
I suspect there is a race condition or something similar in the test suite (or jest) that sometimes causes a failure.  I don't know how to begin to debug that.

**Comment by klassenjs** [comment]
I hit "Restart Build" on the Travis-CI page and the tests came back green, with no commits in between.  Something very strange is going on here.

File: docs/howto/print-layouts.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    mustasche which is the user's inputed title.
+  * `size` - The font size in points.
+  * `fontStyle` - `normal`, `italic` or `bold`
+  * `font` - Defaults to Arial but could be any common font available to the browser.
+  * `color` - Array of 0-255 of `[r, g, b]`.
+* `map` - Put the map on the PDF.
+  * `x` - The left-most side of the map.
+  * `y` - The top most side of the map.
+  * `width` - The width of the map.
+  * `height` - The height of hte map.
```
Comment (klassenjs) [inline docs/howto/print-layouts.md]: hte -> the

File: docs/howto/print-layouts.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+* `text` - Puts text on the map.
+  * `text` - The content to put on the map.
+    Supports [GeoMoose Templates](../templates.md) including a special `{{title}}`
+    mustasche which is the user's inputed title.
+  * `size` - The font size in points.
+  * `fontStyle` - `normal`, `italic` or `bold`
+  * `font` - Defaults to Arial but could be any common font available to the browser.
+  * `color` - Array of 0-255 of `[r, g, b]`.
+* `map` - Put the map on the PDF.
+  * `x` - The left-most side of the map.
```
Comment (klassenjs) [inline docs/howto/print-layouts.md]: What are the units of x,y,...?

File: docs/howto/print-layouts.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  * `y` - The top most side of the map.
+  * `width` - The width of the map.
+  * `height` - The height of hte map.
+* `rect`, `ellipse` - Either shape can be drawn on the PDF.
+  * `x`, `y` - Define the upper left corner of the shape.
+  * `fill` - Array of 0-255 of `[red, green, blue]`. Filled color of the shape.
+  * `stroke` - Array of 0-255 of `[red, green, blue]`. Outline color of the shape.
+  * `width`, `height` - `rect`-only the width and height of the rectangle.
+  * `rx`, `ry` - `ellipse`-only, specifys the x and y radii of the ellipse.
```
Comment (klassenjs) [inline docs/howto/print-layouts.md]: specifies

File: docs/howto/configure-measure-tool.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        services: true,
+        measureToolOptions: {
+            pointProjections: point_projections
+        }
+    });
+```
+
+* `pointProjections` is an array of projection and label definitions,
+  that is defined the same as with the [Coordinate Display](coordinate-display.md).
+  When a user clicks on a point then
```
Comment (theduckylittle) [inline docs/howto/configure-measure-tool.md]: fixed.

File: docs/howto/print-layouts.md (hunk 2 above)
Comment (theduckylittle) [inline docs/howto/print-layouts.md]: I've added a note on units.

**Comment by klassenjs** [comment]
I'm happy with it with my 6b794b9.  If you are happy feel free to merge it.

**Pull request merged**